File: bhaskara.c
#include <stdio.h> 
#include <math.h> 

double calculaX1(int a, int b, int c) {
   double delta = (b*b)-4*(a)*(c); 
   return -b + sqrt(delta) / 2*a; 
} 

double calculaX2(int a, int b, int c) { 
   double delta = (b*b)-4*(a)*(c); 
   return -b - sqrt(delta) / 2*a; 
}
 
int main(int argc, char** argv)  { 
   int a, b, c; 
   printf("Digite a, b e c ... \n"); 
   scanf("%d",&a); 
   scanf("%d",&b);
   scanf("%d",&c); 
   printf(“X1 = %f", calculaX1(a, b, c)); 
   printf(“X2 = %f", calculaX2(a, b, c));
   return 0; 
} 

File: Bhaskara.java
public class Bhaskara { 
   private int a, b, c; 
   
   Bhaskara(int a, int b, int c) { 
      this.a = a; 
      this.b = b; 
      this.c = c; 
    } 

   public double calculaX1() {  
      double delta = (b*b)-4*(a)*(c); 
      return -b + Math.sqrt(delta) / 2*a; 
   } 

   public double calculaX2() { 
     double delta = (b*b)-4*(a)*(c); 
     return -b - Math.sqrt(delta) / 2*a; 
   } 

   public static void main (String args[ ]) { 
      Bhaskara bascara = new Bhaskara(3,-7,2); 
      System.out.println(bascara.calculaX1()); 
      System.out.println(bascara.calculaX2()); 
  } 
} 
